project_task_material_stock/models/project_task.py

```diff
@@ -177,10 +183,29 @@ def _prepare_stock_move(self):
 
     @api.multi
     def create_stock_move(self):
-        for line in self:
-            move_id = self.env['stock.move'].create(
-                line._prepare_stock_move())
-            line.stock_move_id = move_id.id
+        pick_type = self.env.ref('stock.picking_type_out')
+
+        if self:
+            task = self[0].task_id
+            location_id = task.location_source_id.id or task.project_id.location_source_id.id or self.env.ref(
```

Comment:
FYI Here we direct Get Id: `self.ref('stock.stock_location_stock')`

Reply:
@nikul-serpentcs, stock location by default?, if you set an Especific location in project or task?
I think that picking type should be a new, new sequence, etc. (separate them from standard pick types)
What do you think?

Reply:
@jcarlosmontoya If you used this  `self.env.ref('stock.stock_location_stock')` then better to get direct ID like `self.ref('stock.stock_location_stock')`.

Not blocking but improvement

Reply:
@nikul-serpentcs thank you for your tip!